### ISSUE TITLE
UnsupportedOperationException crash from QuadManipulator

### ISSUE THREAD

**Issue by GhostTortilla** [ferreusveritas/Dynamic-Trees#370]
**Describe the bug**
As I am not even remotely close to a developer or programmer I can only follow leads of other people who have encountered a similar problem. The crash-log from this thread seems to be similar to the one I have. And according to the guy responding to the thread the issue is that the mod tries to divide by zero while calculating a color. Afraid I don't have alot more information to give since it crashes while loading the game.
[Link to Forum](https://www.minecraftforge.net/forum/topic/76279-dynamic-trees-crash/)

**Crashes**
- Are you running the latest version of Dynamic Trees?(Currently 0.9.6)
Yes, the version used in this pack is **DynamicTrees-1.12.2-0.9.6.jar**
- If you are using the DT Thaumcraft Add-on do you have the latest version?(Currently 1.4.1e)
The version used in this pack is **DynamicTreesTC-1.12.2-1.4.1e.jar**
Am also using DT Travers Add-on **dttraverse-1.5.jar**

[ModpackCrash.txt](https://github.com/ferreusveritas/DynamicTrees/files/3902149/ModpackCrash.txt)
[Pastebin Version](https://pastebin.com/NwHVP0dL)

Is my first issue written on GitHub, so sorry if anything is not stated properly, will provide extra information if any is needed.

**Comment by kierajreed** [comment]
This is the same crash discussed extensively in #290. It has been since fixed (#345) and is no longer present in the newest version. **Update Dynamic Trees.**

**Comment by Gar0npeX** [comment]
Aint' the same error... one is:
java.lang.UnsupportedOperationException

While the divided by zero is this one:
java.lang.ArithmeticException: / by zero

I don't know about this problem, but I have it out of nowhere since I didnt updated any mods, just CTM and Pickupnotifier (Last one had an error and it's now fixed)

It was working fine Tuesday 26, but next day I launched the game and started throwing that error.

Im using last version of Dynamic trees.

**Comment by Gar0npeX** [comment]
Also there is no update for "DynamicTrees-1.12.2-0.9.7.jar" just 0.9.6. ¿What should I update? I don't understand

**Comment by kierajreed** [comment]
Well in the log on the forum:
> Description: There was a severe problem during mod loading that has caused the game to fail
>
> net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Dynamic Trees (dynamictrees)
> Caused by: java.lang.ArithmeticException: / by zero

**Comment by kierajreed** [comment]
Which is the right log? Run the game again, right now, and pastebin that.

**Comment by Gar0npeX** [comment]
I think is this one

https://pastebin.com/kC5psHJP

**Comment by kierajreed** [comment]
The logs were inconsistent, that's the thing.

**Comment by Gar0npeX** [comment]
> The logs were inconsistent, that's the thing.

I know, maybe GhostTortilla confused one error with another.

It seems this error is pretty recent since It happenened out of nowhere and I didnt modified any files just open the game the next day and it was there.

**Comment by kierajreed** [comment]
Can this error be replicated with only Dynamic Trees? Another mod might be causing this error indirectly.

**Comment by Gar0npeX** [comment]
> Can this error be replicated with only Dynamic Trees? Another mod might be causing this error indirectly.

Will check.

EDIT: It worked! probably another mod then, going to see if I can find what mod is causing problems

**Comment by Gar0npeX** [comment]
"CTM-MC1.12.2-1.0.1.30.jar" is causing the issue.

**Comment by kierajreed** [comment]
Probably manipulating MC model code then. So, unless someone identifies exactly what the issue is, you'll have to choose one or the other. Sorry. (P.S. Thanks for being helpful and testing the issue further. A lot of people don't even do that.)

**Comment by GhostTortilla** [comment]
Thanks for finding a solution for this! @Gar0npeX 
I went into the CTM Config file and enabled the first line. Which resolved the issue. And can now run with earlier mentioned mods again.

> Choose whether the inside corner is disconnected on a CTM block.
> B:connectInsideCTM=true

EDIT: Removing BetterWithMods might have resolved this issue aswell. Will test if that's the case.
Removing Better With Mods & Add-on resolved the issue and many others.
Didn't know about the issue which was resolved earlier, will look more properly into this next time since this might happen more often from now on.
